src/css_table.rs

```diff
@@ -0,0 +1,397 @@
+use std::fmt::Write;
+use super::SYNTAX_SET;
+use std::path::Path;
+use syntect::{
+    html::ClassStyle,
+    parsing::{
+        BasicScopeStackOp, ParseState, Scope, ScopeStack, ScopeStackOp, SyntaxReference, SyntaxSet,
+        SCOPE_REPO,
+    },
+    util::LinesWithEndings,
+};
+
```

Comment:
```suggestion
```

Reply:
Ran it through `rustfmt`, which seemed to fix all these